src/interfaces/pokemonItem.ts

```diff
@@ -0,0 +1,4 @@
+export type pokemonItem = {
```

Comment:
penso che la convenzione dei tipi sia in kebab case, ma forse mi sbaglio 🤔

Reply:
stando alla documentazione di TS, viene usato PascalCase per i tipi e le interfacce. Procedo con la correzione